report/text/template.txt

```diff
@@ -8,7 +8,7 @@ Golang errors in file: [{{ $filePath }}]:
 {{ range $index, $issue := .Issues }}
 [{{ highlight $issue.FileLocation $issue.Severity $issue.NoSec }}] - {{ $issue.RuleID }}{{ if $issue.NoSec }} ({{- success "NoSec" -}}){{ end }} ({{ if $issue.Cwe }}{{$issue.Cwe.SprintID}}{{ else }}{{"CWE"}}{{ end }}): {{ $issue.What }} (Confidence: {{ $issue.Confidence}}, Severity: {{ $issue.Severity }})
 {{ printCode $issue }}
-
+{{ "Proposed Solution" }}: {{ $issue.ProposedSolution }}
```

Comment:
Rename this to `Fix` and skip it when the fix inside of the issue is empty

Reply:
https://github.com/securego/gosec/pull/1177/files#r1700320582